karma.conf.js

```diff
@@ -24,7 +24,8 @@ module.exports = function(config) {
 			// source files, that you wanna generate coverage for
 			// do not include tests or libraries
 			// (these files will be instrumented by Istanbul)
-			'app/scripts/**/*.js': sourcePreprocessors
+			'app/scripts/**/*.js': sourcePreprocessors,
+			'test/spec/**/*.js': sourcePreprocessors
```

Comment:
Hmm interesting point. It would be really bad to have test code that never runs. Opinions aside, that comment is copied from [Karma's Istanbul notes](https://github.com/karma-runner/karma-coverage#basic) and I can't see test inclusion mentioned in [Istanbul's notes on ignoring code](https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md) that you mention in another comment. It also went to 100% even though [Coveralls shows untested lines](https://coveralls.io/builds/3565590/source?filename=app%2Fscripts%2Fangular-apimock.js#L82). So I'm sceptical at the moment =) Either way, that should be a separate PR as it's not related to new functionality.